OpenRA.Mods.Common/Traits/Multipliers/HandicapFirepowerMultiplier.cs

```diff
@@ -0,0 +1,40 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the damage applied by this actor based on the owner's handicap.")]
+	public class HandicapFirepowerMultiplierInfo : TraitInfo
```

Comment:
The descriptions do not make it clear to me if a 5% handicap means "5% less firepower" or "5% firepower".